parsing/src/main/resources/Parser.jj

```diff
@@ -5801,6 +5801,10 @@ SqlNode BuiltinFunctionCall() :
         node = JsonArrayAggFunctionCall() { return node; }
     |
         node = GroupByWindowingCall() { return node; }
+<#list parser.partitionExpressions as method>
```

Comment:
Now that https://github.com/googleinterns/calcite/pull/211 has been merged in, the model we're using for inserting logic is to prefer function overrides to FMPP injection.

Reply:
Changes has made in https://github.com/googleinterns/calcite/pull/212/commits/9361bafe3608b37cfc0cd0c2de5d1a1c4ba31d85 to address the issue,